ros_ign_gazebo_demos/launch/imu.launch.py

```diff
@@ -36,6 +38,14 @@ def generate_launch_description():
         }.items(),
     )
 
+    # RQt
+    rqt = Node(
+        package='rqt_topic',
```

Comment:
you should include this dependency in the package.xml

Reply:
Package `rqt_topic` is already used in some of the other demos, e.g. [`magnetometer.launch.py`](https://github.com/AndrejOrsula/ros_ign/blob/0895c89934c39c3eff290df785cf0487d2502a03/ros_ign_gazebo_demos/launch/magnetometer.launch.py). Therefore, it is already included in [`package.xml`#L27](https://github.com/AndrejOrsula/ros_ign/blob/0895c89934c39c3eff290df785cf0487d2502a03/ros_ign_gazebo_demos/package.xml#L27). Less work for me :smiley:

Reply:
ups, I checked the wrong package.xml